packages/web-components/src/components/rux-dialog/rux-dialog.tsx

```diff
@@ -81,10 +81,29 @@ export class RuxDialog {
     // confirm dialog if Enter key is pressed
     @Listen('keydown', { target: 'window' })
     handleKeyDown(ev: KeyboardEvent) {
-        if (ev.key === 'Enter') {
-            const button = this._getDefaultButton()
-            if (button) {
-                button.click()
+        // prevent this from running if the slots version is being used
+        if (this.open && !this.hasFooter) {
```

Comment:
If the footer is a slot, this 'enter' press listener won't do anything since we don't grab what's being used in that slot. I figured it better to prevent this from running at all in that case.